Grbl_Esp32/src/GCode.cpp

```diff
@@ -194,6 +202,27 @@ Error gc_execute_line(char* line, uint8_t client) {
         // NOTE: Rounding must be used to catch small floating point errors.
         // Check if the g-code word is supported or errors due to modal group violations or has
         // been repeated in the g-code block. If ok, update the command or record its value.
+        
+        if(gc_state.cusGCmdSets)
+        {
+          CustomGCmd *cmdDriver=gc_state.cusGCmdSets->find(letter,int_value,line);
+
+          if(cmdDriver!=NULL)
+          {//do action
+            if(cmdDriver->parse(line, &char_counter)==0)
+            {
+              protocol_buffer_synchronize();
```

Comment:
Is it appropriate to call protocol_buffer_synchronize for all custom GCode commands?

Reply:
I was thinking of that as well, but to dig into the complete cmd flow would be a quite huge modification.
It would be a long term goal to me. 
For the time being, this code can only do simple on point action....

Reply:
I understand - but if we take this patch, when someone tries to use it, they are very likely to run into exactly these issues, and then they will ask us for support.  So by not addressing them, you are turning your problem into our problem.

Reply:
You're right, I will try to find a complete solution.
Thanks for your advising.
Please close this request.